packages/marko-web-identity-x/browser/login.vue

```diff
@@ -167,6 +169,7 @@ export default {
    *
    */
   mounted() {
+    this.loginInputId = `${this.loginInputId}-${this._uid}`;
```

Comment:
The linter doesn't seem to like _uid (causing the Travis build to fail), either add the appropriate eslint-ignore/disable comment to the line prior to this or try using dynamic property access ```this['_uid']```